### ISSUE TITLE
Backend Deployment Script Update

### ISSUE THREAD

**Issue by giancorderoortiz** [SAP/spartacus#7536]
Goal: Update deploy_cx_for_spa.sh script so that it works with 2005 and deploys on linux dev-servers. 

https://github.tools.sap/cx-commerce/c3po-legacy-devconfigs/blob/master/scripts/deploy_cx_for_spa.sh

Tasks (split into tickets?): 
1. Authentication via artifactory.  (ok)
2. Verification of server processes termination (tomcat, solr).  (ok)
3. Retry mechanism. (pending)
4. Confirmation server is up and running (curl swagger?) (ok)
5. Increase Xmx ant-env and project properties. (ok)

Exclusions: patch builds.

**Comment by giancorderoortiz** [comment]
@hackergil is this up to date? https://github.tools.sap/cx-commerce/c3po-legacy-devconfigs/blob/master/spartacus/build.gradle

**Comment by giancorderoortiz** [comment]
Artifactory authentication works. Requires generation of keys for hybris user.

**Comment by giancorderoortiz** [comment]
Depends on #7570

**Comment by giancorderoortiz** [comment]
Working on: https://github.tools.sap/cx-commerce/c3po-legacy-devconfigs/blob/feature/GH-7537/scripts/deploy_cx_2005_for_spa.sh

**Comment by giancorderoortiz** [comment]
Added a script for restarting backend in case of failure... https://github.tools.sap/cx-commerce/c3po-legacy-devconfigs/blob/feature/GH-7537/scripts/backend_ping_restart.sh

**Comment by giancorderoortiz** [comment]
pr here https://github.tools.sap/cx-commerce/c3po-legacy-devconfigs/pull/23/files

**Comment by giancorderoortiz** [comment]
Request by @hackergil to import impex script to enable users in order to allow orders to complete.

**Comment by giancorderoortiz** [comment]
Request to use correct version of spartacussampledata (i.e. 2005/latest)

**Comment by giancorderoortiz** [comment]
Request to import apparel test data as well. 
i.e. spartacussampledataaddon/resources/spartacussampledataaddon/import/stores/apparel-uk/test-data.impex

**Comment by giancorderoortiz** [comment]
pr updated per latest request. 
Most issues in pr addressed.

**Comment by giancorderoortiz** [comment]
waiting for cr

**Comment by giancorderoortiz** [comment]
Out of scope for now. Re-open if necessary
